cmd/lncli/commands.go

```diff
@@ -3362,6 +3363,11 @@ var updateChannelPolicyCommand = cli.Command{
 			Usage: "the CLTV delta that will be applied to all " +
 				"forwarded HTLCs",
 		},
+		cli.Uint64Flag{
+			Name: "max_htlc_msat",
+			Usage: "if set, the max HTLC size that will be applied " +
+				"to all forwarded HTLCs",
```

Comment:
Does not setting mean that there is no max or that the max is left unchanged? Comment could be explicit about this.